pyall/analysis-2016_02/meta.yaml

```diff
@@ -1,23 +1,25 @@
+{% set version = "2016_02" %}
+
 package:
   name: analysis
-  version: '2016_01'
+  version: {{ version }}
 
 build:
-  number: 3
+  number: 0
 
 requirements:
   run:
-    - python
-    - jupyter
-    - ipywidgets
-    - pymongo
-    - datamuxer
-    - databroker
+    - bluesky  # [py3k]
     - channelarchiver
+    - databroker
+    - datamuxer
+    - ipywidgets
+    - jupyter
     - lmfit ==0.8.3
     - matplotlib
     - openpyxl
+    - pymongo
```

Comment:
IIRC we were pinning it previously.  Now we *might* not need to pin?  Or maybe we do need to pin at `pymongo >=3.2`?  If we don't need to pin, I agree this can go away